application/src/main/java/org/opentripplanner/standalone/config/routerequest/RouteRequestConfig.java

```diff
@@ -544,7 +545,7 @@ duration can be set per mode(`maxDurationForMode`), because some street modes se
               Safety limit to prevent access to and egress from too many stops.
               """
               )
-              .asInt(dftAccessEgress.defaultMaxStopCount()),
+              .asInt(MaxStopCountLimit.DEFAULT_LIMIT),
```

Comment:
I think this should something like this. Otherwise the default from the config doesn't work here
```suggestion
              .asInt(dftAccessEgress.maxStopCountLimit().defaultLimit()),
```